Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/Dictionary.java

```diff
@@ -0,0 +1,7 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+public interface Dictionary {
```

Comment:
Even though this interface only has one method, I still think it's a good idea because it means that the code can be easily modified to support multiple dictionaries, and the classes that use the dictionary are not coupled to `ArrayDictionary`